examples/integrations/itk/SelectROI.ipynb

```diff
@@ -0,0 +1,350 @@
+{
```

Comment:
<p>This gif is wicked cool to have embedded in the notebook!! How did you do that? :-D</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/InsightSoftwareConsortium/itkwidgets/pull/703/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
It's just an html image tag in a markdown cell! Updated to use the full URL instead of the relative path though so it may appear broken in the review notebook now until these changes are merged in.